### DISCUSSION TITLE
Add branch protection rule: require conversation resolution before merging PR

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#10225] [Suggest an Idea]
There's a new branch protection rule, its in _beta_, but seems very helpful. 😉 

> - [ ] **Require conversation resolution before merging**
> When enabled, all conversations on code must be resolved before a pull request can be merged into a branch that matches this rule.

I noticed we sometimes merge PRs when there are still discussions that have not been explicitly resolved. I think this rule will prevent some cases where we "need to redo part of the PR because of missed comment(s)".

You can find this new setting by going to the Renovate repository -> settings tab - branches sidebar -> click edit next to `main` branch -> search for **Require conversation resolution before merging**.

GitHub documentation for new feature: https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-conversation-resolution-before-merging

**Comment by rarkins** [comment]
Done
